### PR TITLE
[Hotfix][docs] Fixed errors references for  'site.scala version suffix' and 'site.version'

### PR DESCRIPTION
**Pull request by RocMarshal**
## What is the purpose of the change

*Fixed errors references for  'site.scala version suffix' and 'site.version'*

## Brief change log

*Fixed errors references for  'site.scala version suffix' and 'site.version'*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (**not applicable** / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by RocMarshal** [comment]
As a follow-up to https://github.com/apache/flink/pull/16300#issuecomment-869337797

**Comment by RocMarshal** [comment]
Could someone please help me to advance this PR ? Thank you.

**Comment by RocMarshal** [comment]
@alpinegizmo Could you please help me to check it ? Thank you.

**Comment by RocMarshal** [comment]
@JingsongLi Could you please help me to check it ? Thank you.

File: docs/content.zh/docs/deployment/resource-providers/standalone/docker.md [hunk 1]
```diff
@@ -323,7 +323,7 @@ RUN ln -s /usr/bin/python3 /usr/bin/python
 
 # install Python Flink
 {{< stable >}}
-RUN pip3 install apache-flink[=={{site.version}}]
+RUN pip3 install apache-flink[=={{version}}]
```
Comment (sjwiesman) [inline docs/content.zh/docs/deployment/resource-providers/standalone/docker.md:326]:
```suggestion
RUN pip3 install apache-flink[=={{< version >}}]
```

**Review by sjwiesman — COMMENTED** [review]
One issue I'll fix when merging. 

In the future, if you are not already working with a committer, please open a ticket and find one to assign it who can then help review the PR instead of tagging people after you've started the work. That will ensure you receive a timely review and the change is seen by the right person.

**Comment by sjwiesman** [comment]
merged in master and release-1.13

**Pull request closed without merging**